pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/ImageVenueUploaderSection/VenueImageUploaderModal.tsx

```diff
@@ -122,6 +130,8 @@ export const VenueImageUploaderModal = ({
           onEditedImageSave={onEditedImageSave}
           onReplaceImage={onReplaceImage}
           onSetCredit={setCredit}
+          scale={scale}
+          updateScale={updateScale}
```

Comment:
```suggestion
          updateInitialScale={setEditorInitialScale}
```
J'ai l'impression qu'on à pas besoin du passe-plat `updateScale`